client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -636,8 +635,8 @@ export class Checkout extends React.Component {
 		}
 
 		return (
-			<div className="main main-column" role="main">
```

Comment:
There's probably global-level styling for `.main`, tho it's a bit unusual that this `className` would be used directly here -- typically, I think this is used by the `<Main />` component. One consideration would be to replace this `<div />` with such a component.

It's a bit weird to have two nested `<div />`s here anyway (where the parent doesn't have any other children). Maybe one `<Main className="checkout__main">` would be enough here. (That would also allow us to avoid a `className` like `checkout__checkout`, which is rather suboptimal IMO.)

Reply:
I think you're right here @ockham.

If I recall correctly, the linter allows the directory-based "namespace" to be used directly in a few files, one of them is `main`. If we rename this file to main (if it is a sort of main and that makes sense), then we can do something like this directly, drop the inner `div`, and the class names may not need to be touched:

```jsx
<Main className="checkout">
```